### PR TITLE
[W6.1e][W09-A4]Charmaine Lee

### PR DESCRIPTION
**Pull request by charmaineleehc**

### CODE REVIEW

**Comment by goweiwen** [comment]
Hi Charmaine, please update your PR title to reflect your phase B team. Thanks!

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -43,4 +43,22 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
```
Comment (eugenepeh) [inline test/java/seedu/addressbook/common/UtilsTest.java:48]: These tests can be separated into multiple more specific methods. Refer to https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html for our standard of naming the methods.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -43,4 +43,22 @@ private void assertAreUnique(Object... objects) {
...
+        assertNoNull();
+        assertNoNull(1, 2, 3);
+        assertNoNull("a", "b", "c");
+
+        assertHasNull(null, 2, 3);
+        assertHasNull("a", null, null);
```
Comment (eugenepeh) [inline test/java/seedu/addressbook/common/UtilsTest.java:54]: Well organised and efficient test cases, can try to include newly instantiate object such as `new Object()` too

**Review by eugenepeh — COMMENTED** [review]
Good job on completing the activity.
Some comments added. Please close the PR after reading the comments.

**Pull request closed without merging**